sdk/iot/README.md

```diff
@@ -1,14 +1,38 @@
-# Azure IoT Clients for Embedded C
+# Embedded C for Azure IoT Clients
 
-Official Embedded C clients for Azure IoT Hub and Device Provisioning Service.
+Official Embedded C clients for Azure IoT Hub services.
 
 ## Getting started
 
-TBD
+- **Docs**: If you're new to the Azure IoT Embedded C SDK take a look at the [Machine State diagram](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/iot/doc/mqtt_state_machine.md#device-provisioning-service) that explains the high-level architecture, SDK components, and a clear view of SDK x Application responsibilities. 
+
+- **Build**: The Azure IoT library is compiled following the same steps listed on the root [README](https://github.com/Azure/azure-sdk-for-c/blob/master/README.md) documentation, under ["Getting Started Using the SDK"](https://github.com/Azure/azure-sdk-for-c/blob/master/README.md#getting-started-using-the-sdk). 
+
+
+- **Samples**: [This page](https://github.com/Azure/azure-sdk-for-c/tree/master/sdk/iot/hub/samples) explains samples for the Azure Embedded C SDK IoT Hub and [this page](https://github.com/Azure/azure-sdk-for-c/tree/master/sdk/iot/provisioning/samples) shows the Provisioning Clients and how to use them.
 
 ### Prerequisites
 
-TBD
+For compiling the Azure IoT Embedded C SDK for the most common platforms (Windows and Linux), no further prerequisites are necessary.
+Please follow the instructions in the Getting Started section above.
+For compiling for specific target devices, please refer to their specific toolchain documentation. 
+
+## Key Features
+
+
+Feature | Azure IoT Embedded C SDK | Description

```

Comment:
This is great work here.

I have a nit to make it a little easier on the eyes. In the [current SDK](https://github.com/Azure/azure-iot-sdk-c#device-client-sdk), we have nice green checkmarks for supported, the gray x for "not yet supported", and the dash for no support planned. I think it would look good if we did something similar here.

So for where `[Supported]` is used, you can use `:heavy_check_mark:` which would look like ✔️
for `[Partially Supported]` we can use `:heavy_check_mark:*` or ✔️*
and for `[Not supported yet]` we can use `:heavy_multiplication_x:` or ✖️

Reply:
and at the top would be a breakdown of what each means.